tool/teleport/common/integration_configure.go

```diff
@@ -218,6 +220,32 @@ func onIntegrationConfAccessGraphAWSSync(ctx context.Context, params config.Inte
 	return trace.Wrap(awsoidc.ConfigureAccessGraphSyncIAM(ctx, iamClient, confReq))
 }
 
+func onIntegrationConfAzureOIDCCmd(ctx context.Context, params config.IntegrationConfAzureOIDC) error {
+	// Ensure we print output to the user. LogLevel at this point was set to Error.
+	utils.InitLogger(utils.LoggingForDaemon, slog.LevelInfo)
+
+	appID, tenantID, err := azureoidc.SetupEnterpriseApp(ctx, params.ProxyPublicAddr, params.AuthConnectorName)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	if params.AccessGraph {
+		err := azureoidc.CreateTAGCacheFile(ctx)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+	}
+
+	fmt.Println()
+	fmt.Println("Success! Use the following information to finish the integration onboarding in Teleport:")
+	fmt.Printf("Tenant ID: %s\nClient ID: %s\n", tenantID, appID)
+	if params.AccessGraph {
+		fmt.Println("Because Access Graph integration was enabled, we also created a file with the necessary data.")
```

Comment:
I don't think that the fact that access graph is enabled is important. The important part is that something else needs to be done to finish the setup.
```suggestion
		fmt.Println("To finish the setup you will need `cache.json` file that we created for you.")
```